### PR TITLE
project paths fixes

### PR DESCRIPTION
**Pull request by olexandrd**
- css links
- vite build fix

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,56 +1,57 @@
...
+      rel="stylesheet"
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link href="./css/main.css" rel="stylesheet" />
+    <title>Your Energy</title>
+    <link rel="icon" type="image/x-icon" href="./favicon.svg" />
+  </head>
+  <body>
+    <header class="header">
```
Comment (LadaM) [inline src/index.html:30]: Why is the space changed? Is it formatted with prettier?

Comment (olexandrd) [reply src/index.html:30]: It was formatted with prettier

Comment (olexandrd) [reply src/index.html:30]:
```
"INFO" - 20:25:28] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used
["INFO" - 20:25:28] Prettier Options:
{
  "filepath": "/Users/study/code/advanced-js/goit-advancedjs-team-3/src/index.html",
  "parser": "html",
  "useTabs": false,
  "tabWidth": 2,
  "printWidth": 80,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "es5",
  "bracketSpacing": true,
  "arrowParens": "avoid",
  "proseWrap": "always"
}
["INFO" - 20:25:28] Formatting completed in 9ms.
```

**Review by LadaM — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (LadaM) [reply src/index.html:30]: In my local version of prettier file I see this:

```
{
  "printWidth": 80,
  "useTabs": false,
  "semi": true,
  "singleQuote": true,
  "trailingComma": "es5",
  "bracketSpacing": true,
  "arrowParens": "avoid",
  "proseWrap": "always"
}
```

**Pull request merged**